waffle/management/commands/waffle_flag.py

```diff
@@ -29,7 +30,8 @@ def add_arguments(self, parser):
             action='store',
             type=int,
             dest='percent',
-            help='Roll out the flag for a certain percentage of users. Takes a number between 0.0 and 100.0'),
+            help='Roll out the flag for a certain percentage of users. Takes '
```

Comment:
Thanks for fixing all these up!

Reply:
For sure! I noticed there are other Flake8 errors throughout the code. I'll open a PR to fix these in the next week or so.